### ISSUE TITLE
NPM Package Version Not Showing the latest version

### ISSUE THREAD

**Issue by tonesto7** [hoobs-org/HOOBS#409]
**Description**
When trying to install the homebridge-smartthings-v2 plugin it shows the older version 2.0.10 version instead of the latest v2.1.4.  
Once it's installed I need to reboot Hoobs for it to show me the update option to update to the latest version.

**Version**
Please include the version of HOOBS you are using.
3.1.19
**Did you upgrade**
Please let us know if you upgraded from a previous version.
- [x] Yes
- [ ] No

**Pervious version**
If you upgraded, please let us know your previous version.

**Did you orginally upgrade to HOOBS 3 from HOOBS 2.1.1?**
Did you perform an upgrade from HOOBS 2.1.1? Some things are different.
- [ ] Yes
- [x] No

**What device are you using?**
Please let us know the device you are running HOOBS on.
- [ ] HOOBS Box
- [x] HOOBS SD Card
- [ ] Custom Install

**If this is a custom install, what's your operating system?**
Please let us know what operating system and version you are using. Ex. Fedora 30 or macOS Catalina.

**List your plugins**
Please include a list of the plugins you are using. We need to be able to find it on NPM. If the plugin is no published, please include a link to the repository.
homebridge-smartthings-v2

**Post your config**
This can be found in the interface, Configuration -> Advanced or you can get it via SSH `cat ~/.hoobs/etc/config.json`

* Remove any passwords from the config file. This is public.

```
POST CONFIG HERE
```

**Post your log**
You can get the log from the intercafe. This is in the Log section.

```
POST LOG HERE
```

**Additional context**

**Comment by mkellsy** [comment]
I noticed that it takes a bit of time for NPM to get the new version propagated to API that HOOBS uses. This version was only released 35 minutes ago. I will give it some time.

Then on top of that there is a 30 minute cache in HOOBS. We do this to stop so many calls to NPM. I don't want them to block us :)

A tall tell sign is the readme is not there yet when you go to the Details, but the version is correct.
1. The search uses the /search command
2. Details uses the /package command

**Comment by mkellsy** [comment]
In the meantime, If you need to test the new version. Here's how I do it.
1. Make sure the plugin is installed at any version, from the UI
2. Run `npm pack` on the plugin project, this will create a .tgz file
4. Upload that .tgz file to HOOBS `scp ./homebridge-smartthings-v2-[version].tgz hoobs@hoobs.local:~/` I use the home directory.
5. Run this on HOOBS `cd ~/.hoobs/` then `npm install ~/homebridge-smartthings-v2-[version].tgz`

**Comment by mkellsy** [comment]
I just ran this https://registry.npmjs.org/-/v1/search?text=homebridge-smartthings-v2+keywords:homebridge-plugin&size=50. It is directly from NPM and the version reported is. 2.0.10.

Here's the result
```json
{
    "objects": [
        {
            "package": {
                "name": "homebridge-smartthings-v2",
                "scope": "unscoped",
                "version": "2.0.10",
                "description": "SmartThings plugin for HomeBridge",
                "keywords": [
                    "homebridge-plugin",
                    "smartthings",
                    "homekit",
                    "homebridge"
                ],
                "date": "2019-12-17T17:38:24.477Z",
                "links": {
                    "npm": "https://www.npmjs.com/package/homebridge-smartthings-v2",
                    "homepage": "https://github.com/tonesto7/homebridge-smartthings-v2/#readme",
                    "repository": "https://github.com/tonesto7/homebridge-smartthings-v2",
                    "bugs": "http://github.com/tonesto7/homebridge-smartthings-v2/issues"
                },
                "publisher": {
                    "username": "tonesto7",
                    "email": "tonesto7@gmail.com"
                },
                "maintainers": [
                    {
                        "username": "tonesto7",
                        "email": "tonesto7@gmail.com"
                    }
                ]
            },
            "score": {
                "final": 0.26525344618367114,
                "detail": {
                    "quality": 0.47265755899371387,
                    "popularity": 0.01939860519682947,
                    "maintenance": 0.3333333333333333
                }
            },
            "searchScore": 100000.23
        }
    ],
    "total": 1,
    "time": "Wed Jan 08 2020 23:50:39 GMT+0000 (UTC)"
}
```

I think you need to open a support ticket with NPM  
https://www.npmjs.com/support

**Comment by tonesto7** [comment]
I published a new version last night (2.1.7) and using your query appears to be returning the correct version now.

**Comment by csprocket777** [comment]
What made the difference here? I'm running in to the same issue with my plugin which isn't updating in the plugin repo in the hoobs app.

`homebridge-cisco-collaboration-devices`
